pkg/generated/bindata.go

```diff
@@ -324,6 +478,153 @@ func translationsKubectlEn_usLc_messagesK8sPo() (*asset, error) {
+
+func translationsKubectlFr_frLc_messagesK8sMoBytes() ([]byte, error) {
+	return _translationsKubectlFr_frLc_messagesK8sMo, nil
+}
+
+func translationsKubectlFr_frLc_messagesK8sMo() (*asset, error) {
+	bytes, err := translationsKubectlFr_frLc_messagesK8sMoBytes()
+	if err != nil {
+		return nil, err
+	}
+
+	info := bindataFileInfo{name: "translations/kubectl/fr_FR/LC_MESSAGES/k8s.mo", size: 0, mode: os.FileMode(0), modTime: time.Unix(0, 0)}
+	a := &asset{bytes: bytes, info: info}
+	return a, nil
+}
+
+var _translationsKubectlFr_frLc_messagesK8sPo = []byte(`# Test translations for unit tests.
+# Copyright (C) 2016
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR brendan.d.burns@gmail.com, 2016.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: gettext-go-examples-hello\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2013-12-12 20:03+0000\n"
+"PO-Revision-Date: 2017-01-27 21:04-0800\n"
+"Last-Translator: Brendan Burns <brendan.d.burns@gmail.com>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.6.10\n"
+"X-Poedit-SourceCharset: UTF-8\n"
+"Language-Team: \n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"Language: fr\n"
+
+msgctxt "Update the annotations on a resource"
+msgid "Update the annotations on a resource"
+msgstr "Mettre à jour les annotations sur une ressource"
```

Comment:
`sur` kind of feels weird to me in this context because I literally interpret it as `onto`. 

Instead, I would use the following, which is a bit closer to `of`:
```
Mettre à jour les annotations d'une ressource
```